themes/vokuro/users/index.volt

```diff
@@ -3,6 +3,7 @@
 <div class="mb-5">
     {{ link_to("users/create", "Create Users", "class": "btn btn-primary") }}
 </div>
+<h6 id="adminDemoContainer">Seperated admin javascript has been loaded to private/admin layout</h6>
```

Comment:
Where did you saw `private/admin` ?

Reply:
we call it private, but I think at the end it is admin area. And I think that everybody cloning the project removes all of those files asap after cloning the repo. My idea is that to be visible before someone delete the files. 
I can change is to `users` context as It is the module name if you prefer it?

Reply:
I would just don't add that line. :)